test/chart/BarChart.spec.tsx

```diff
@@ -14,6 +14,19 @@ describe('<BarChart />', () => {
 
   type CustomLabelProps = Partial<{ x: number; y: number; index: number }>;
 
+  test('Renders 1 bar in simple BarChart', () => {
+    const { container } = render(
+      <BarChart width={800} height={200} data={[{ x: 1, y: 200 }]}>
+        <XAxis type="number" dataKey="x" />
+        <YAxis />
+        <Tooltip />
+        <Bar dataKey="y" />
+      </BarChart>,
+    );
+
+    expect(container.querySelectorAll('.recharts-rectangle')).toHaveLength(1);
```

Comment:
can you confirm that this test doesn't pass before your change? If the rectangle gets rendered with a 0 width this would still pass

Reply:
Really, this condition can be satisfied in this case if the rectangle renders with a 0 width. I will verify.

Reply:
The .recharts-rectangle isn't rendered when its width is zero, then, this test fails without my changes.